### DISCUSSION TITLE
Fixing issues uncovered during reconciliation

### DISCUSSION THREAD

**Discussion by BackSeat** [firefly-iii/firefly-iii#8161] [Firefly III]
I'm trying out Firefly's reconciliation feature.

I reconciled an account, but there was one payment missing in Firefly. It had the same date and amount as another payment so it had been identified as a duplicate and omitted, although in this case it wasn't a duplicate.

First issue: I'm 90% through the reconciliation. If I leave it now to manually enter the missing transaction, I'll lose all the reconciliation done so far. What's the best way of dealing with this next time?

Second issue: when I clicked on "Store reconciliation", the dialog box offered me the option of "Clear the selected transactions, and create a correction adding £3.29 to this asset account." At the bottom of the dialog box it said, "You can always edit or delete a correction later."

So that sounded like the easiest option: let Firefly create a correction, and then I can edit it to be what it should be. So I did that, and Firefly created a reconciliation entry.

I found it and tried to edit it but I find I can't edit the source or destination account, and I did want to change the destination account.

My question: what is the best way to correct issues found during a reconciliation?

**Comment by JC5** [comment]
> First issue: I'm 90% through the reconciliation. If I leave it now to manually enter the missing transaction, I'll lose all the reconciliation done so far. What's the best way of dealing with this next time?

Why would you lose all reconciliation so far?

> I found it and tried to edit it but I find I can't edit the source or destination account, and I did want to change the destination account.

You can edit the amount, not the accounts. You can't reconcile one account and then connect the correction to another account. That makes zero sense to me.

**Comment by BackSeat** [comment]
> Why would you lose all reconciliation so far?

I assumed that if I navigated away from the reconciliation page I'd lose the checked checkboxes (but I didn't test that).

> You can edit the amount, not the accounts. You can't reconcile one account and then connect the correction to another account. That makes zero sense to me.

Quite. However, the missing transaction was to pay vendor X, and the correction didn't let me assign the amount to that vendor. Maybe I've misunderstood.

**Reply by JC5** [reply]
> I assumed that if I navigated away from the reconciliation page I'd lose the checked checkboxes (but I didn't test that).

Try it with one checkbox. You'll notice you'd be fine. 😉 

> Quite. However, the missing transaction was to pay vendor X, and the correction didn't let me assign the amount to that vendor. Maybe I've misunderstood.

Then you'll have to create a transaction and remove the reconciliation. The system is strict about that.
